.github/ISSUE_TEMPLATE/pt2_bug_report.yml

```diff
@@ -0,0 +1,49 @@
+name: 🐛 torch.compile Bug Report
+description: Create a report to help us reproduce and fix the bug
+labels: ["oncall: pt2"]
+
+body:
+  - type: markdown
+    attributes:
+      value: >
+        #### Before submitting a bug, please make sure the issue hasn't been already addressed by searching through [the
+        existing and past issues](https://github.com/pytorch/pytorch/issues)
+        It's likely that your bug will be resolved by checking our FAQ or troubleshooting guide [documentation](https://pytorch.org/docs/master/dynamo/index.html)
+  - type: textarea
+    attributes:
+      label: 🐛 Describe the bug
+      description: |
+        Please provide a clear and concise description of what the bug is.
+      placeholder: |
+        A clear and concise description of what the bug is.
+    validations:
+      required: false
+
+  - type: textarea
+    attributes:
+      label: Error logs
+      description: |
+        Please provide the error you're seeing
+      placeholder: |
+        Error...
+    validations:
+      required: false
+  - type: textarea
+    attributes:
+      label: Minified repro
+      description: |
+        Please run the minifier on your example and paste the minified code below
+        Learn more here https://pytorch.org/docs/master/dynamo/troubleshooting.html
+      placeholder: |
+        env TORCHDYNAMO_REPRO_AFTER="aot" python your_model.py
+        or
+        env TORCHDYNAMO_REPRO_AFTER="dynamo" python your_model.py
+
+        import torch
+        ... 
+
+        # torch version: 1.13.....
+
+        class Repro(torch.nn.Module)
+    validations:
+      required: false
```

Comment:
Do you want to add the same section as the regular one for getting version through the collect_env script? https://github.com/pytorch/pytorch/blame/9c6433ce48c27f4b6ecf585bf6049176c217bbbe/.github/ISSUE_TEMPLATE/bug-report.yml#L41-L52

Also should we collect pt2 related info in that script (like compiler, triton version, nvcc available, etc)?

Reply:
Yeah good idea